README.md

```diff
@@ -2,14 +2,17 @@
 
 This repository contains a collection of API clients for easy access to the Tributech DataSpace Kit.
 
+## Available Api Clients
+The following clients are available:
+
+| Framework / Language | Package | Release version | Usage | Examples |
+|-|-|-|-|-|
+| .NET Core / C# | Nuget: [Tributech.Dsk.Api.Clients](https://www.nuget.org/packages/Tributech.Dsk.Api.Clients) | ![Release version](https://img.shields.io/nuget/v/Tributech.Dsk.Api.Clients) | [Usage](./clients/netcore) | [Example](./examples/netcore)
+
 ## Integration via APIs
 The clients can be used to create a custom connector to connect a data source to the relevant Tributech Dataspace Kit endpoints. The clients help in achieving the goal of integrating data sources via the Tributech Dataspace Kit APIs, namely the Data Api and Trust Api, as seen in the image below (marked with the red rectangle).
 
 ![Integration via APIs](./assets/img/integration.jpg)
-
```

Comment:
I think we should describe the difference between Data API and Trust API and in which case which API should be used. I would suggest to briefly explain the following cases:

Store values (suggestion to always use Trust APIs /value(s) operations if no Agent is used)
Get values (hint for sync state)
Store Proofs
Get Proofs
Validate Proofs

Reply:
To be honest I would place such explanations about when to use what Apis / general understanding of the product in the docs.tributech.io, and focus here on the clients themselves and how to use them. 
But to be pragmatic we could add it here for now and relocate later if necessary / wanted since it's always just markdown. 

Will add it and notify you @patricklamplmair